hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -410,50 +680,48 @@
-                  count:    "NWkupDetect",
-                  swaccess: "rw",
-                  hwaccess: "hro",
-                  regwen:   "REGWEN",
-                  cname:    "DETECTOR",
+    { multireg: { name:         "WKUP_DETECTOR",
+                  desc:         "Configuration of wakeup condition detectors."
+                  count:        "NWkupDetect",
+                  compact:      "false",
+                  swaccess:     "rw",
+                  hwaccess:     "hro",
+                  regwen:       "WKUP_DETECTOR_REGWEN",
+                  regwen_multi: "true",
+                  cname:        "DETECTOR",
                   fields: [
                     { bits: "2:0",
                       name: "MODE",
                       resval: 0,
                       desc: "Wakeup detection mode."
                       enum: [
                         { value: "0",
-                          name: "Disabled",
-                          desc: "Pin wakeup detector is disabled."
+                          name: "Posedge",
+                          desc: "Trigger a wakeup request when observing a positive edge."
                         },
                         { value: "1",
                           name: "Negedge",
                           desc: "Trigger a wakeup request when observing a negative edge."
                         },
                         { value: "2",
-                          name: "Posedge",
-                          desc: "Trigger a wakeup request when observing a positive edge."
-                        },
-                        { value: "3",
                           name: "Edge",
                           desc: "Trigger a wakeup request when observing an edge in any direction."
                         },
-                        { value: "4",
-                          name: "TimedLow",
+                        { value: "3",
+                          name: "TimedHigh",
                           desc: '''
-                            Trigger a wakeup request when pin is driven LOW for a certain amount
+                            Trigger a wakeup request when pin is driven HIGH for a certain amount
                             of always-on clock cycles as configured in !!WKUP_DETECTOR_CNT_TH.
                             '''
                         },
-                        { value: "5",
-                          name: "TimedHigh",
+                        { value: "4",
+                          name: "TimedLow",
                           desc: '''
-                            Trigger a wakeup request when pin is driven HIGH for a certain amount
+                            Trigger a wakeup request when pin is driven LOW for a certain amount
                             of always-on clock cycles as configured in !!WKUP_DETECTOR_CNT_TH.
                             '''
                         },
+
                       ]
                     }
                     { bits: "3",
```

Comment:
FYI: @tjaychen this is the register I have been talking about in the other comment